soc/nxp/kinetis/Kconfig.defconfig

```diff
@@ -3,14 +3,6 @@
 
 if SOC_FAMILY_KINETIS
 
-config SERIAL_INIT_PRIORITY
-	default 55
-	depends on SERIAL
-
-config ADC_INIT_PRIORITY
```

Comment:
This does not seem to be related to this commit, should we not change this priority?

Reply:
it doesn't seem necessary, since it looks like ADC already initializes after DMA

Reply:
I suspect this has the same reasoning as the SERIAL.

Reply:
But digging into history and it looks like the initial PR from 2020 set it to this value

Reply:
updated the commit message